src/main.js

```diff
@@ -2,22 +2,25 @@
 
 'use strict';
 
-require = require('esm')(module);
+import fs from 'fs';
```

Comment:
```suggestion
import fs from 'node:fs';
```


Maybe we can add (idk what this is called, namespace?) namespaces?

Reply:
Is that preferred? I'm not to familiar with that, but it would be a simple change. i think that exists in a few files.